src/components/author/revision/checkout-reject-buttons.tsx

```diff
@@ -58,7 +65,7 @@ export function CheckoutRejectButtons({
         <FontAwesomeIcon icon={faCheck} className="lg:mr-0.5" />{' '}
         {strings.revisions.checkout.action}
       </button>
-      {!isRejected && (
+      {!isRejected && !isPage && (
```

Comment:
pages are not part of the revision process and therefore don't need to be rejected